### PR TITLE
Weav 240 clean up timestamps within response object

### PR DESCRIPTION
**Pull request by mathieub93**

### CODE REVIEW

File: src/main/java/com/weaverplatform/postgresql/database/query/model/QueryResult.java [hunk 1]
```diff
@@ -63,37 +62,64 @@ public void setCount(int count) {
   }
 
   public long getExecutionTimeStart() {
-    return executionTimeStart;
+    if (getTimes().containsKey("executionTimeStart"))
```
Comment (bastbijl) [inline src/main/java/com/weaverplatform/postgresql/database/query/model/QueryResult.java]: could you still use {}s although it is not strictly necessary? we always to that to keep it a little more readable

File: src/main/java/com/weaverplatform/postgresql/database/query/model/QueryResult.java [hunk 2]
```diff
@@ -63,37 +62,64 @@ public void setCount(int count) {
...
+      return getTimes().get("executionTimeStart");
+    return 0;
   }
 
   public long getProcessingTimeEnd() {
-    return processingTimeEnd;
+    if (getTimes().containsKey("processingTimeEnd"))
```
Comment (bastbijl) [inline src/main/java/com/weaverplatform/postgresql/database/query/model/QueryResult.java]: idem

File: src/main/java/com/weaverplatform/postgresql/database/query/model/QueryResult.java [hunk 3]
```diff
@@ -63,37 +62,64 @@ public void setCount(int count) {
...
+  public void setSubqueryTime(long subqueryTime) {
+    setTimes("subqueryTime", subqueryTime);
+  }
+
+  public HashMap<String, Long> getTimes() {
+    return times;
+  }
+
+  public void setTimes(String time, Long duration) {
+    if (times.containsKey(time))
```
Comment (bastbijl) [inline src/main/java/com/weaverplatform/postgresql/database/query/model/QueryResult.java]: idem

File: pom.xml [hunk 4]
```diff
@@ -15,7 +15,7 @@
     <dependency>
       <groupId>com.weaverplatform</groupId>
       <artifactId>sdk-java</artifactId>
-      <version>3.2.2</version>
+      <version>3.2.0</version>
```
Comment (gvdent) [inline pom.xml]: Dependency going backward? Is this intentional?

**Comment by bastbijl** [comment]
Question: the response object now looks like this

{
  timeA: nnn,
  timeB: nnn,
  timeC: nnn,
  nodes: {
    x, y, z
  }
}

could you change it into

{
  nodes: {
    x, y, z
  }
  times: {
    timeA: nnn,
    timeB: nnn,
    timeC: nnn
  }
}

File: pom.xml (hunk 4 above)
Comment (mathieub93) [reply pom.xml]: No this is not intended, didn't do anything with the sdk-java. No idea where this came from.

**Pull request merged**